templates/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -0,0 +1,225 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local base = import "base.libsonnet";
+local timeouts = import "../../timeouts.libsonnet";
+local tpus = import "../../tpus.libsonnet";
+local utils = import "../../utils.libsonnet";
+
+{
+  local command_common = |||
+    git clone https://github.com/huggingface/transformers.git
```

Comment:
Would it be better to create a new image that has this repository or add it to our PyTorch image?

Reply:
I did initially do this here https://github.com/pytorch/xla/pull/1892 but based on comments, will clone as part of the test.